tests/baselines/reference/emptyArrayBindingPatternParameter05.js

```diff
@@ -0,0 +1,26 @@
+//// [tests/cases/conformance/es6/destructuring/emptyArrayBindingPatternParameter05.ts] ////
+
+//// [emptyArrayBindingPatternParameter05.ts]
+export const cilBlurLinear: string[][] = [[]];
+
+const [,] = cilBlurLinear;
+
+let [,] = cilBlurLinear;
+
+var [,] = cilBlurLinear;
+
+const [[]] = cilBlurLinear;
+
+let [[]] = cilBlurLinear;
+
+var [[]] = cilBlurLinear;
+
+
+//// [emptyArrayBindingPatternParameter05.js]
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.cilBlurLinear = void 0;
+exports.cilBlurLinear = [[]];
+var _a = exports.cilBlurLinear[0];
+var _b = exports.cilBlurLinear[0];
+var _c = exports.cilBlurLinear[0];

```

Comment:
we can see here that `const [[]] = ...` isn't as "optimized" as `const [,] = ...`. I think it's fine-ish. All this code assumes `downlevelIteration: false` so it's OK to drop those element access expressions but it's also kinda OK to keep them. It would be cool to use a consistent logic here but that's a separate issue in my eyes

Reply:
I'd prefer we didn't just entirely elide the destructuring given proxies/getters - but I'll defer to others. @RyanCavanaugh @rbuckton